File: swe4302_lab/Lab_03/VehicleMS/src/Vehicle.java
public class Vehicle {

    public String name;


    public Vehicle(String name) {
        this.name = name;
    }

    public void Start() {
        System.out.println(name + " vehicle started");
    }

    public void Stop() {
        System.out.println(name + " vehicle stopped");
    }
}

File: swe4302_lab/Lab_03/C_Sharp_VehicleMS/VehicleMS/Vehicle.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace VehicleMS
{
    public class Vehicle
    {
        public string Name { get; set; }

        public Vehicle(string name)
        {
            Name = name;
        }

        public void Start()
        {
            Console.WriteLine(Name + " vehicle started");
        }

        public void Stop()
        {
            Console.WriteLine(Name + " vehicle stopped");
        }
    }
}
